python/social_bot/tasks.py

```diff
@@ -104,11 +105,11 @@ class GoalTask(Task):
 
     def __init__(self,
                  env,
-                 max_steps=200,
+                 max_steps,
                  goal_name="ball",
                  distraction_list=['coke_can', 'table', 'car_wheel', 'plastic_cup', 'beer'],
                  success_distance_thresh=0.5,
-                 fail_distance_thresh=0.5,
```

Comment:
0.5 suites for simple navigation but is too small for playground. slightly moving backwards the goal ends the episode.